src/Land/Model/SoilWaterParameterizations.jl

```diff
@@ -532,4 +542,67 @@ function matric_potential(model::BrooksCorey{FT}, S_l::FT) where {FT}
     return ψ_m
 end
 
+
+
+"""
+    inverse_matric_potential(
+        model::vanGenuchten{FT},
+        ψ::FT
+    ) where {FT}
+
+Compute the effective saturation given the matric potential, using
+the van Genuchten formulation.
+"""
+function inverse_matric_potential(model::vanGenuchten{FT}, ψ::FT) where {FT}
+
+    ψ > 0 && error("Matric potential is positive")
+
+    n = model.n
+    m = model.m
+    α = model.α
```

Comment:
In case you'd prefer to write this as `@unpack_fields model n m α`, I'm going to move `@unpack_fields` (in the thermo test suite) to its own module in ClimateMachine (I may use it myself elsewhere, too).